src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -779,20 +750,13 @@ void Chttp2ServerListener::OnAccept(void* arg, grpc_endpoint* tcp,
     MutexLock lock(&self->mu_);
     // Shutdown the the connection if listener's stopped serving.
     if (!self->shutdown_ && self->is_serving_) {
-      if (!grpc_resource_user_safe_alloc(channel_resource_user,
-                                         GRPC_RESOURCE_QUOTA_CHANNEL_SIZE)) {
```

Comment:
Is this removed now because we can overcommit memory against the quota? This looked like an effective (but maybe blunt) way to provide backpressure. Might the removal open us up to trying to accept a flood of connections and crashing when we otherwise would not have?